tests/opentelemetry/conftest.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+import opentelemetry
+import pytest
+
+from ddtrace._opentelemetry import TracerProvider
+
+
+TRACER_PROVIDER = TracerProvider()
+# set_tracer_provider can only be called once
+opentelemetry.trace.set_tracer_provider(TRACER_PROVIDER)
```

Comment:
revisit this

Reply:
Using subprocess tests requires a lot of duplication and makes the tests harder to read. I will punt on this for now.